pkg/pcr/measurement.go

```diff
@@ -48,6 +48,19 @@ func (chunk DataChunk) String() string {
 	return string(b)
 }
 
+// Find returns the chunk with the specified DataChunkID.
+// Or returns nil if such data chunk was not found.
+func (s DataChunks) Find(id DataChunkID) *DataChunk {
+	for idx := range s {
+		d := &s[idx]
+		if d.ID == id {
+			return d
+		}
+	}
+
+	return nil
+}
+
```

Comment:
Was required while testing if IBB is indeed verified correctly (in our internal code).